app/libs/loaders/Leaf_autoloader.php

```diff
@@ -93,7 +108,7 @@
 $oc_config = new Orgchart\Config($site_paths, $oc_settings);
 if (!defined('ORGCHART_CONFIG')) define('ORGCHART_CONFIG', $oc_config);
 
-ini_set('session.gc_maxlifetime', 2592000);
+//ini_set('session.gc_maxlifetime', 2592000);

```

Comment:
Is this something we wanted to do or was this from testing changes? Is this setup in the docker configs?

Reply:
This has been in there. When I added the session_start to the top of this page this errored so I commented it out. I would assume we probably want to add it to the docker config or somewhere if this is really what we want.